src/balance.rs

```diff
@@ -81,6 +82,49 @@ pub enum LightningBalance {
 		/// The amount available to claim, in satoshis, excluding the on-chain fees which will be
 		/// required to do so.
 		amount_satoshis: u64,
+		/// The transaction fee we pay for the closing commitment transaction. This amount is not
+		/// included in the `amount_satoshis` value.
+		///
+		/// Note that if this channel is inbound (and thus our counterparty pays the commitment
+		/// transaction fee) this value will be zero. For channels created prior to LDK Node 0.4
+		/// the channel is always treated as outbound (and thus this value is never zero).
```

Comment:
> the channel is always treated as outbound (and thus this value is never zero).

Not sure what is meant by this. Is because LDK Node only allowed outbound channels? If so, why do we need to say this?